### DISCUSSION TITLE
Centos 7 GUI install feedback

### DISCUSSION THREAD

**Discussion by gerberla** [Chia-Network/chia-blockchain#1498] [General]
Thought I'd drop in my experience installing on centos 7 since it's not as common. The install guide was pretty well put together, and the commands for install seemed to work (yum and script execution). I ran into few problems. The first was chome-sandbox installed in node_modules. I had to run `sudo chown root node_modules/electron/dist/chrome-sandbox` and `sudo chmod 4755 node_modules/electron/dist/chrome-sandbox`. Not sure if there was something else I could have done to avoid that. The easiest problem was needing ffmpeg installed. I'm sure different installations have various missing libs depending on the centos installation options. This was just mine. :)

The last was more user error (I laughed at myself after I found it out) related and lack of understanding of how the electron app worked. When calling `npm run electon`, you need access to the system gui (duh). I was trying it from putty and the logs weren't entirely helpful besides showing a SIGABRT. I didn't try it, but I'm sure if I had setup an xserver it might have worked. Anyways, great job! I'm excited to see how this goes.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
